src/components/BackEndRolePage/BackEndRolePage.js

```diff
@@ -2,26 +2,31 @@ import React from 'react';
 import { Route } from 'react-router-dom';
 
 import { BACK_END_ROLE_PATH, BRAG_PATH, CONFESS_PATH } from '../../constants/routes';
-import { BRAG_VARIANT, CONFESS_VARIANT } from '../../constants/coloursVariants';
-import WideButton from '../WideButton';
+import { BRAG_ROLE_ACTION, CONFESS_ROLE_ACTION } from '../../constants/roles';
+import RoleActionButton from '../RoleActionButton';
 import RoleBar from '../RoleBar';
 import Section from '../Section';
 import QuoteBubble from '../QuoteBubble';
 import Default from './Default';
 import Brag from './Brag';
 import Confess from './Confess';
 
+// TODO Replace with auto-generated message @blurbyte
+const QUOTE = 'Yesterday I was implementing PUT method for missing RESTful API endpoint.';
+
 function BackEndRolePage() {
   return (
     <>
-      <RoleBar />
+      <RoleBar label="Back End Developer" />
       <Section>
         <Route exact path={BACK_END_ROLE_PATH} component={Default} />
         <Route exact path={BACK_END_ROLE_PATH + BRAG_PATH} component={Brag} />
         <Route exact path={BACK_END_ROLE_PATH + CONFESS_PATH} component={Confess} />
-        <QuoteBubble />
-        <WideButton variant={BRAG_VARIANT}>Brag about my efforts</WideButton>
-        <WideButton variant={CONFESS_VARIANT}>Confess my mistake</WideButton>
+        <QuoteBubble quote={QUOTE} />
+        <nav>
```

Comment:
Why can't we just stay with the previous way of rendering buttons and don't create unnecessary abstraction over that. We can, and we should simplify that in other way.

Since all of that button are rendered in two groups:
- `RoleButtons`
- `ActionButtons`

Why can't we enclose the `WideButton` (and name it a bit better) inside these button groups, that can handle proper containing (`nav`) and take care about proper routes for each button.
The approach that you propose cost us a lot of effort:
- Adds more components to maintain
- Forces us to perform conditions inside these components
- Forces us to handle additional exceptions

...and the gain is only predefined Icon and proper colour.

Still we get more complicated rendering API. We just replace:
```
<WideButton variant={BRAG_VARIANT}>Brag about my efforts</WideButton>
<WideButton variant={CONFESS_VARIANT}>Confess my mistake</WideButton>
```

with:
```
<RoleActionButton variant={BRAG_ROLE_ACTION} to={BACK_END_ROLE_PATH + BRAG_PATH} />
<RoleActionButton variant={CONFESS_ROLE_ACTION} to={BACK_END_ROLE_PATH + CONFESS_PATH} />
```

IMO this doesn't improve anything. If we spot the issue with the way how we handle buttons right now, i am happy to propose some easier way to handle that.

Reply:
Just trying to figure out how to merge roles & paths since they are rather identical.

Reply:
The problem with previous solution was it had too many moving parts imho. Here if you want to add more roles / action buttons you just operate in single directory.

Reply:
Also we created colorVariants constants with was unnecesarry intertop over design tokens, when in fact for more clarity you can use composition and apply styles directly. But I agree this RoleActionButton API is messy ATM and trying to solve in in next commit :)